symbols/overlay29.yml

```diff
@@ -1304,6 +1538,17 @@ overlay29:
         
         r0: user entity pointer
         r1: target entity pointer
+    - name: IsBlinded
+      address:
+        NA: 0x23177E4
+        EU: 0x2318244
+        JP: 0x2318CB4
+      description: |-
+        Returns true if the monster has the blinded status, or if it is not the leader and is holding Y-Ray Specs. If the second parameter is false, skips checking for the held item.
```

Comment:
Maybe this should reference `status_id::STATUS_BLINKER`?

Reply:
Couldn't hurt I guess. I'm less concerned with symbols than I am with struct fields since it should be easy enough to just go look at the symbol address to see the correspondence.

Reply:
Would something like this be ok?
```
Returns true if the monster has the blinded status, which is represented by status_id::STATUS_BLINKER (statuses::blinded set to 1). [...]
```

Reply:
That's a bit wordy. `statuses::blinded` already has a comment referencing `STATUS_BLINKER`, so let's just write:

> Returns true if the monster has the blinded status (see statuses::blinded). [...]